test/acceptance/features/bindAppToService.feature

```diff
@@ -7,183 +7,256 @@ Feature: Bind an application to a service
     Background:
         Given Namespace [TEST_NAMESPACE] is used
         * Service Binding Operator is running
-        * PostgreSQL DB operator is installed
+        * OLM Operator "backend" is running
```

Comment:
is this step needed at given that in the next step we apply right CRD?

Reply:
Removed it.